### PR TITLE
fix(plugin): check whether dataTemplate is an empty field

### PR DESCRIPTION
**Pull request by elpsyr**
Signed-off-by: crui <crui@relaper.com>

### CODE REVIEW

File: extensions/sinks/influx/influx.go [hunk 1]
```diff
@@ -76,26 +76,28 @@ func (m *influxSink) Configure(props map[string]interface{}) error {
...
 	}
 	if i, ok := props["fields"]; ok {
 		if i, ok := i.(string); ok {
 			m.fields = i
 		}
 	}
-	if _, ok := props["dataTemplate"]; ok {
-		m.hasTransform = true
+	if i, ok := props["dataTemplate"]; ok {
+		if i.(string) != "" {
```
Comment (ngjaying) [inline extensions/sinks/influx/influx.go:98]: i.(string) may panic if it is not a string, shall we add the check also?

Comment (elpsyr) [reply extensions/sinks/influx/influx.go:98]: @ngjaying You are right. We need to judge whether this is a string first.I will perfect it.

**Review by ngjaying — APPROVED** [review]
Thank you!

**Pull request merged**